### DISCUSSION TITLE
Weather condition example

### DISCUSSION THREAD

**Discussion by amiler** [HASwitchPlate/openHASP#362] [Hardware]
I am using **LOLIN TFT 2.4 shield** with **ESP32-WROVER (PSram)** and I am trying a weather forecast example shown under Home Assistant integration. The main icon in the top left corner does not display (128 pixel size). It displays "no data" even though the text description of the current condition is displayer properly. If image is scaled to something smaller (96 pix for example), icon displays properly. Smaller icons (32 pix) are all fine.
Anybody having similar problem? 

Also, I have selected D1 pin on the shield for backlight cntl. What is the pin that needs to be configured under Display Settings to be able to control backlight via PWM? Is it GPIO25 (that maps to D1 on ESP32 D1 mini)?

**Comment by amiler** [comment]
For backlight - It seems that it works if you set any GPIO other than "none". It does not have to be specific GPIO that maps to D1 pin. 

For icons - still do not have a clue why 128x128p icons are not loading and any other size below that is fine. There is enough free pSram memory

**Reply by rjblake** [reply]
@amiler - did you ever fix this? I have exactly the same problem and cannot find any reason why it doesn't work. I didn't try scaling images, but will give this a try as well.
